src/main/web/js/app/accessibility-enhance.js

```diff
@@ -21,5 +21,5 @@ function highchartsAccessibilityAttrs(selector, labelText, removeAttrs) {
 }
 
 function timeseriesAccessibilityAttrs(removeAttrs) {
-    highchartsAccessibilityAttrs($('.timeseries__chart'), 'Chart representing data available in table alternative. Select "table" in filters to display table', removeAttrs);
+    highchartsAccessibilityAttrs($('.timeseries__chart'), 'Chart representing data available. Select "table" in show data as to display data as a table', removeAttrs);
```

Comment:
Do we have copy for this? I think it's OK as is, but I do wonder if the bit specifically, "In show data as to display data as a table", can be made clearer that it's referring to the fieldset above

Reply:
Good point. I'll raise this with Ben.